frontend/src/components/project/ProjectSliderWithTitle.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Typography, Divider } from "@material-ui/core";
+import ProjectsSlider from "../climateMatchResults/ProjectsSlider";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => {
+  return {
+    subHeader: {
+      fontWeight: "bold",
+      paddingBottom: theme.spacing(2),
+    },
+    divider: {
+      marginTop: theme.spacing(1),
+      marginBottom: theme.spacing(1),
+    },
+  };
+});
+
+export default function ProjectSliderWithTitle({ similarProjects, title }) {
```

Comment:
What's the advantage of an additional wrapper component over adding a `title` prop to `ProjectsSlider` directly? I'm not sure yet what's better so I'm wondering what your thought process was.

Reply:
At first, I had just added a Typography + ProjectsSlider component to every tab but noticed I'm doing the same thing 3 times so I wanted to move it into a new reusable component, but I agree, adding a title prop to the slider might make more sense.